currencies.yml

```diff
@@ -0,0 +1,1544 @@
+---
+NIO:
+  minor_unit: 2
+  currency: Cordoba Oro
+  symbol: C$
+  tags:
+  - :iso4217
```

Comment:
I think it would actually be good, seeing it now, to have all the current active ISO4217 codes back in `iso4217.yml`. We can then introduce a `iso4217-historical.yml` to represent currencies that were ISO4217 but are no longer active (replaced by others). Further, we should have an `unofficial.yml`, which would contain any currency we want to support (such as crypto) that isn't part of the ISO4217 standard. This could also include any arbitrary currency that's not officially recognised, some of which are listed on Wikipedia (https://en.wikipedia.org/wiki/ISO_4217#Currencies_without_ISO_4217_currency_codes), such as the Cook island dollar (pegged to NZD).